pages/content/amp-dev/about/websites-2021.html

```diff
@@ -0,0 +1,110 @@
+---
+$title: AMP Websites
+$view: /views/2021/base.html
+description: "AMP is a simple and robust format to ensure your website is fast, user-first, and makes money. AMP provides long-term success for your web strategy with distribution across ..."
+sharing_images:
+  square: /static/img/sharing/about-websites-1x1.png
+  wide: /static/img/sharing/about-websites-600x314.png
+---
+
+<main>
+  <section class="ap-stage">
+    <div class="ap-stage-text">
+      <div class="ap-text-block">
+        <h2>AMP Websites</h2>
+        <h1>{{ _('A framework that amplifies page experience.') }}</h1>
+        <p>
+          {{ _('AMP is a simple and robust web components framework that makes
+          it easy to build fast websites that are user-first and makes money.
+          AMP provides long-term success for your web strategy with distribution
+          across popular platforms and reduced operating costs.') }}
+        </p>
+
+        <a class="ap-button --inverted"
+          href="{{ g.doc('/content/amp-dev/documentation/index.html', locale=doc.locale).url.path }}">
+          {{ _('Get started with AMP') }}
+        </a>
+      </div>
+    </div>
+
+    <div class="ap-stage-backdrop">
+      <svg width="457" height="457">
+        <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/static/frontend/sprite.svg#deco-foundation"></use>
+      </svg>
+    </div>
+
+    <div class="ap-stage-media">
+      <amp-img src="/static/img/about/websites-amp-sidebar.jpg" layout="responsive" width="653" height="567"
+        alt="{{ _('A screenshot showing amp-sidebar.') }}"></amp-img>
+    </div>
+  </section>
+
+  <section class="ap-section --col-3-22">
+    <div class="ap-text-media">
+      <div class="ap-text-media-col">
+        {% include 'views/2021/partials/extras/page-experience/pie-chart.j2' %}
+      </div>
+      <div class="ap-text-media-col">
+        <div class="ap-text-block">
+          <h2 class="ap-h1">{{ _('Guaranteed Page Experience') }}</h2>
+          <p>
+            {{ _('AMP provides you with the tools needed to build websites that
+            meet Page Experience ranking signals while remaining flexible and
+            customizable to your needs. Page experience is a set of signals from
+            Google that measure how users perceive their experience interacting
+            with a web page beyond its pure information value. AMP is a
+            straightforward way to a high page experience score. Domains built
+            with AMP are 5 times more likely to pass* Core Web Vitals metrics
+            than non-AMP!') }}
```

Comment:
Would you be able to add these strings to the [localization files](https://github.com/ampproject/amp.dev/tree/future/pages/translations), so they can be picked up automatically for localization?

Reply:
Done.

Reply:
Awesome! Does grow recognize it still with the whitespace being used here? I am foggy on if it can or not

Reply:
To be save I took them out.